src/client.js

```diff
@@ -4530,6 +4712,96 @@ module.exports.CRYPTO_ENABLED = CRYPTO_ENABLED;
  *     perform the key verification
  */
 
+/**
+ * Fires when private keys for cross-signing need to be saved.
+ * @event module:client~MatrixClient#"cross-signing.savePrivateKeys"
+ * @param {object} keys the private keys to save.
+ * @param {UInt8Array} [keys.master] the private master key
+ * @param {UInt8Array} [keys.self_signing] the private user-signing key
+ * @param {UInt8Array} [keys.user_signing] the private self-signing key
+ */
+
+/**
+ * Fires when a private key is needed.
+ * @event module:client~MatrixClient#"cross-signing.getKey"
+ * @param {object} data
+ * @param {string} data.type the type of key needed.  Will be one of "master",
+ *     "self_signing", or "user_signing"
+ * @param {Function} data.done a function to call with the private key as a
+ *     `UInt8Array`
+ * @param {Function} data.cancel a function to call if the private key cannot
+ *     be provided
+ * @param {string} [data.error] Error string to display to the user.  Normally
```

Comment:
We'll have to i18n these I'm afraid so we should define some constants for them instead.